python_legacy/tests/api/test_conversions.py

```diff
@@ -79,3 +87,9 @@ def test_to_bytes(self):
                          Literal.of(uuid.UUID("f79c3e09-677c-4bbd-a479-3f349cb785e7")).to_byte_buffer())
         self.assertEqual(b'foo', Literal.of(bytes(b'foo')).to_byte_buffer())
         self.assertEqual(b'foo', Literal.of(bytearray(b'foo')).to_byte_buffer())
+        # Decimal on 2-bytes
+        self.assertEqual(b'\x30\x39', Literal.of(123.45).to(DecimalType.of(5, 2)).to_byte_buffer())
+        # Decimal on 3-bytes to test that we use the minimum number of bytes
+        self.assertEqual(b'\x12\xd6\x87', Literal.of(123.4567).to(DecimalType.of(7, 4)).to_byte_buffer())
+        # Negative decimal to test two's complement
+        self.assertEqual(b'\xed\x29\x79', Literal.of(-123.4567).to(DecimalType.of(7, 4)).to_byte_buffer())
```

Comment:
Can you make sure the test cases are shared between the Java implementation and here? There's a test for Java in `TestConversions` but it would be good to add these cases and ensure that they match. I'd also like to have a case that requires an empty byte, like `Literal.of(0.11).to(DecimalType.of(10, 2)).to_byte_buffer()`, as well as one for negative numbers.

Reply:
@rdblue Should I remove the "old" tests in python and only keep the ones in-sync with Java?
I also added 3 tests in the Java implementation as requested.

Reply:
No, I think keeping all the tests is a good thing. If anything, we should probably add the Python tests to Java.

Reply:
Oh, looks like you already did!

Reply:
Yes, I added 3 tests to the Java implementation that I felt were not covered by the Java test suite.  If it's what you are talking about...